front/app/translations/admin/es-CL.json

```diff
@@ -321,9 +321,11 @@
   "app.containers.Admin.Insights.components.TopBar.renameModalSaveView": "Renombrar",
   "app.containers.Admin.Insights.components.TopBar.renameModalTitle": "Crear análisis",
   "app.containers.Admin.Insights.createModalCancel": "Cancelar",
+  "app.containers.Admin.Insights.createModalCollapse": "Colapso",
   "app.containers.Admin.Insights.createModalDescription": "Estás a punto de tener tus primeros análisis sobre lo que los participantes están hablando. Estas percepciones se guardarán para que puedas seguir trabajando en ellas más adelante.",
+  "app.containers.Admin.Insights.createModalExpand": "Ampliar",
   "app.containers.Admin.Insights.createModalNameLabel": "Nombre de la tendencia",
-  "app.containers.Admin.Insights.createModalProjectScopeLabel": "Selecciona un proyecto",
+  "app.containers.Admin.Insights.createModalProjectsLabel": "Project(s)",
```

Comment:
@Brent This looks like regression. Was Spanish, now English.

Reply:
It's also changed in German, etc.

Reply:
It's a new translation that hasn't been translated yet. You can see it's new because the id has changed from `app.containers.Admin.Insights.createModalProjectScopeLabel` to `app.containers.Admin.Insights.createModalProjectsLabel`. You can verify this by searching for createModalProjectsLabel in the codebase, then you'll see this is in a messages.ts file and is used in the front-end (in a .tsx file). The other (old) one, can't be found in such files anymore.

However, since it's not translated yet, it should not have been in crowdin_master, so the feature branch this is coming from didn't have the required translations yet. I'll post a message about this.